spring-example-user/src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+spring:
+  application:
+    name: federation-jvm-spring-example
```

Comment:
Shouldn't each app have its own name? Otherwise, we should be able to share resources like this across examples using a common module.

Reply:
Yes this is a miss, all apps should have individual Spring application names.